docs/application/tizen-studio/web-tools/web-inspector.md

```diff
@@ -1,3 +1,15 @@
+Check the following prerequisites before attempting to debug Web applications using Chrome.
+
+##Chrome Requirements
+The Web Inspector tool is displayed in a new Google Chrome™ browser window.
+To use the web inspector tool, you need to install Chrome™ browser in your on your system.
+Following the below steps.
+Go to Window > Preferences > Tizen Studio > Web > Chrome.
+The Google Chrome™ browser location is automatically set if the browser is installed on your system.
+If you have installed the browser in a custom location, the Tizen Studio cannot automatically detect the path and you must set the location manually as shown below.
+![Chrome setting](./media/setchrome_webinspector.png)
+
+
 # Debugging with Web Inspector
```

Comment:
The page title is "# Debugging with Web Inspector". So please move the added part under this line.

Also, below is my suggestion for the doc flow (in rough) : 
@surbhig12 would you please ask the reviewer who will check this PR, to check the below suggestion also?

# Debugging with Web Inspector

You can [debug Web applications](../../web/tutorials/process/run-debug-app.md) using the JavaScript Debugger tool. The JavaScript Debugger is based on Webkit Web Inspector, and has been modified to support remote debugging.

**Figure: JavaScript Debugger panels**

![JavaScript Debugger panels](./media/jsdebugger_panels.png)

When debugging with the [emulator](../common-tools/emulator.md), the emulator communicates with the Google Chrome&trade; browser through the HTTP protocol.

When the JavaScript Debugger is started, the **Network** panel is off. To enable the **Network** panel and start monitoring the resource loading status, press the F5 key. This reloads the current page and displays the load time on the **Network** panel.

## "Prerequisites" or "Chrome requirements"

The Web Inspector tool is displayed in a new **Google Chrome™** browser window.
To use the web inspector tool, you must install **Chrome™** browser in your on your system as below steps:
Go to **Window** > **Preferences** > **Tizen Studio** > **Web** > **Chrome**.
The **Google Chrome™** browser location is automatically set, if the browser is already installed.
If the browser installed in a custom location, Tizen Studio cannot automatically detect the path and the location must be set the manually.
![Chrome setting](./media/setchrome_webinspector.png)

## Elements Panel
...